core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -51,1729 +50,1729 @@
-    assertEquals("date(DATE '2020-08-17')", expr.toString());
-
-    expr = DSL.date(DSL.literal(new ExprDateValue("2020-08-17 12:12")));
-    assertEquals(DATE, expr.type());
-    assertEquals(new ExprDateValue("2020-08-17 12:12"), eval(expr));
-    assertEquals("date(DATE '2020-08-17')", expr.toString());
-  }
-
-  @Test
-  public void day() {
-    FunctionExpression expression = DSL.day(DSL.literal(new ExprDateValue("2020-08-07")));
-    assertEquals(INTEGER, expression.type());
-    assertEquals("day(DATE '2020-08-07')", expression.toString());
-    assertEquals(integerValue(7), eval(expression));
-
-    expression = DSL.day(DSL.literal("2020-08-07"));
-    assertEquals(INTEGER, expression.type());
-    assertEquals("day(\"2020-08-07\")", expression.toString());
-    assertEquals(integerValue(7), eval(expression));
-  }
-
-  @Test
-  public void dayName() {
-    FunctionExpression expression = DSL.dayname(DSL.literal(new ExprDateValue("2020-08-07")));
-    assertEquals(STRING, expression.type());
-    assertEquals("dayname(DATE '2020-08-07')", expression.toString());
-    assertEquals(stringValue("Friday"), eval(expression));
-
-    expression = DSL.dayname(DSL.literal("2020-08-07"));
-    assertEquals(STRING, expression.type());
-    assertEquals("dayname(\"2020-08-07\")", expression.toString());
-    assertEquals(stringValue("Friday"), eval(expression));
-  }
-
-  @Test
-  public void dayOfMonth() {
-    FunctionExpression expression = DSL.dayofmonth(
-        functionProperties, DSL.literal(new ExprDateValue("2020-08-07")));
-    assertEquals(INTEGER, expression.type());
-    assertEquals("dayofmonth(DATE '2020-08-07')", expression.toString());
-    assertEquals(integerValue(7), eval(expression));
-
-    expression = DSL.dayofmonth(functionProperties, DSL.literal("2020-07-08"));
-    assertEquals(INTEGER, expression.type());
-    assertEquals("dayofmonth(\"2020-07-08\")", expression.toString());
-    assertEquals(integerValue(8), eval(expression));
-  }
-
-  private void testDayOfMonthWithUnderscores(FunctionExpression dateExpression, int dayOfMonth) {
-    assertEquals(INTEGER, dateExpression.type());
-    assertEquals(integerValue(dayOfMonth), eval(dateExpression));
-  }
-
-  @Test
-  public void dayOfMonthWithUnderscores() {
-    FunctionExpression expression1 = DSL.dayofmonth(
-        functionProperties, DSL.literal(new ExprDateValue("2020-08-07")));
-    FunctionExpression expression2 = DSL.dayofmonth(functionProperties, DSL.literal("2020-07-08"));
-
-    assertAll(
-        () -> testDayOfMonthWithUnderscores(expression1, 7),
-        () -> assertEquals("dayofmonth(DATE '2020-08-07')", expression1.toString()),
-
-        () -> testDayOfMonthWithUnderscores(expression2, 8),
-        () -> assertEquals("dayofmonth(\"2020-07-08\")", expression2.toString())
+    final List<DateFormatTester> dateFormatTesters = ImmutableList.of(
+            new DateFormatTester("1998-01-31 13:14:15.012345",
```

Comment:
git diff goes crazy with indentation changes. Please, revert to remove it.

Reply:
Formatting has been fixed